src/EFCore.PG/Update/Internal/NpgsqlModificationCommandBatch.cs

```diff
@@ -97,9 +110,8 @@ protected override void Consume(RelationalDataReader reader)
                     // Find the next propagating command, if any
                     int nextPropagating;
                     for (nextPropagating = commandIndex;
-                        nextPropagating < ModificationCommands.Count &&
-                        !ModificationCommands[nextPropagating].RequiresResultPropagation;
-                        nextPropagating++) ;
+                         nextPropagating < ModificationCommands.Count && !ModificationCommands[nextPropagating].RequiresResultPropagation;
+                         nextPropagating++) {}
```

Comment:
Indent probably changed by autoformat. By the way, thee common way is to have a semicolon instead an empty body.

Reply:
Even if it's common....should it be? This is a fairly non-standard use of the `for`-block in the C# world (as opposed to C++ where this appears commonly).

I tend to agree with the [inspection explanation](https://www.jetbrains.com/help/rider/EmptyEmbeddedStatement.html) from JetBrains on this.

As for the indenting, that was an auto-format, so I can revert...butL

```c#
for (int i = 0;
     i < 10;
     i ++)
{
// ...
}



```